crowbar_framework/config/repos.yml

```diff
@@ -0,0 +1,173 @@
+---
+suse-12.0:
+  repos:
+    sles12-pool:
+      name: "SLES12-Pool"
+      required: "mandatory"
+      product_name: "common"
+      content_key:
+      repomd:
+        key:
+        tag: "obsproduct://build.suse.de/SUSE:SLE-12:GA/SLES/12/POOL/x86_64"
+        md5: "9a62177e1c6852d48453ed3909956d6b"
+      patterns: []
+      url:
+      remote:
+    sles12-updates:
+      name: "SLES12-Updates"
+      required: "recommended"
```

Comment:
Should be mandatory, we don't support things when SLE updates are missing (as we depend on fixes in updates).